### DISCUSSION TITLE
Welcome to googleAnalyticsR Discussions!

### DISCUSSION THREAD

**Discussion by MarkEdmondson1234** [8-bit-sheep/googleAnalyticsR#345] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baikulov** [comment]
Hey!

I have a problem with function **google_analytics** and customDimensions. I get a maximum of 50,000 lines. In fact, there are many more of them in GA.

Example request:
ga_stat_query <- google_analytics (ga_id $ viewId,
                                  date_range = c (startDate, endDate),
                                  metrics = c ("sessions"),
                                  dimensions = c ("dimension12"),
                                  max = -1)

How do I fix this?

Thank!

**Reply by MarkEdmondson1234** [reply]
I moved your question to the issues as this thread is more about general questions not bug fixes https://github.com/MarkEdmondson1234/googleAnalyticsR/issues/374
